apps/pie-docs/package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "@11ty/eleventy": "1.0.2",
     "@11ty/eleventy-navigation": "0.3.5",
+    "@justeattakeaway/pie-icons": "3.2.0",
```

Comment:
This isn't a dev dependency though, is it?

Reply:
Is that a case for not using directly the workspace?

Reply:
@fernandofranca Just replicating the version number that was being used previously.

Having it hardcoded also has the advantage of not being forced to refactor consumers right away when a change is made. Something we need to agree within the team I think on how to handle deps like this.

@webbertakken Ah yeah, good spot! Force of habit. Will update :D